src/mango_selector_text.erl

```diff
@@ -171,11 +171,12 @@ convert(_Path, {Props} = Sel) when length(Props) > 1 ->
 
 
 to_query({op_and, Args}) when is_list(Args) ->
-    Res = ["(", join(<<" AND ">>, lists:map(fun to_query/1, Args)), ")"],
+    QueryArgs = lists:map(fun to_query/1, Args),
+    Res = ["(", mango_util:join(<<" AND ">>, QueryArgs), ")"],
```

Comment:
Don't assign to a variable if you're just returning that variable directly. Remove the `Res =` and just make this the last line of this clause.